test/common/scale_up_replicas_rhsso.go

```diff
@@ -14,20 +15,36 @@ import (
 )
 
 var (
-	numberOfRhssoReplicas     int = 2
-	numberOfUserRhssoReplicas int = 3
-	scaleDownRhssoReplicas    int = 1
-	rhssoName                     = "rhsso"
-	rhssoNamespace                = NamespacePrefix + "rhsso"
-	userSSOName                   = "rhssouser"
-	userSSONamespace              = NamespacePrefix + "user-sso"
-	requestURLSSO                 = "/apis/keycloak.org/v1alpha1"
-	kindSSO                       = "Keycloaks"
+	numberOfRhssoReplicas     = 2
+	numberOfUserRhssoReplicas = 3
+	rhssoName                 = "rhsso"
+	userSSOName               = "rhssouser"
+	requestURLSSO             = "/apis/keycloak.org/v1alpha1"
+	kindSSO                   = "Keycloaks"
 )
 
-func TestReplicasInRHSSOAndUserSSO(t *testing.T, ctx *TestingContext) {
-	checkScalingOfKeycloakReplicas(t, ctx, rhssoName, GetPrefixedNamespace("rhsso"), numberOfRhssoReplicas)
-	checkScalingOfKeycloakReplicas(t, ctx, userSSOName, GetPrefixedNamespace("user-sso"), numberOfUserRhssoReplicas)
+func TestReplicasInRHSSO(t *testing.T, ctx *TestingContext) {
+	inst, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatalf("failed to get RHMI instance %v", err)
+	}
+	if resources.RunningInProw(inst) {
+		checkScalingOfKeycloakReplicas(t, ctx, userSSOName, GetPrefixedNamespace("user-sso"), 1)
```

Comment:
```suggestion
		checkScalingOfKeycloakReplicas(t, ctx, rhssoName, GetPrefixedNamespace("rhsso"), 1)
```
Should this not be RHSSO instead ? :thinking: